src/Ticket.php

```diff
@@ -3277,6 +3290,20 @@ public static function getSpecificValueToDisplay($field, $values, array $options
         switch ($field) {
             case 'type':
                 return self::getTicketTypeName($values[$field]);
+            case '_virtual_age':
+                $calendars_id = Entity::getUsedConfig('id', $values['entities_id'], 'calendars_id', 0);
+
+                if ($calendars_id) {
+                    $calendar = new Calendar();
+                    $calendar->getFromDB($calendars_id);
+                    $time = $calendar->getActiveTimeBetween($values['date'], $_SESSION["glpi_currenttime"]);
+                } else {
+                    $ticket_date = new DateTime($values['date']);
+                    $now = new DateTime($_SESSION["glpi_currenttime"]);
+                    $time = $now->getTimestamp() - $ticket_date->getTimestamp();
+                }
+
+                return sprintf(__('%s hours %s minutes ago'), floor($time / 3600), floor(($time % 3600) / 60));
```

Comment:
The `ago` does not seems relevant here. Indeed, we are rendering a duration that does not reflect the real time, so we cannot say "16 hours ago" for a ticket created 2 days ago on a calendar that has 8 working hours per day.